client/src/utils/playlist.js

```diff
@@ -96,4 +96,24 @@ export const removeFavoritePlaylist = (playlistId) => {
       "x-access-token": localStorage.getItem('token')
     }
   })
+}
+
+export const addSimilarityMethodCount = (similarityMethod) => {
```

Comment:
When is this called? 🤔

Reply:
It's called in UserTrack.jsx. I think it did not show up b/c this pr got messed up...